YAML/YAML.sublime-syntax

```diff
@@ -513,51 +515,48 @@ contexts:
 
   flow-pair:
     - match: \?
-      scope: punctuation.definition.key-value.begin.yaml
+      scope: meta.mapping.key.explicit.yaml punctuation.definition.key.begin.yaml
       push:
-        - meta_scope: meta.flow-pair.explicit.yaml
-        - match: (?=[},\]]) # Empty mapping keys & values are allowed
-          pop: true
-        - include: flow-pair
-        - include: flow-node
-        - match: :(?=\s|$|{{c_flow_indicator}})
-          scope: punctuation.separator.key-value.mapping.yaml
-          set: flow-pair-value
-    # Attempt to match plain-in scalars and highlight as "entity.name.tag",
-    # if followed by a colon
+        - - match: ''
```

Comment:
This multi-push might not be needed if we'd move on to sublime-syntax version 2.

Note: Setting `version: 2` doesn't break any of the existing tests.

Otherwiese: 

1. With inheritance in the house it might be a better choice/style for basic syntaxes to avoid anonymous contexts.
2. AFAIK we are asked to avoid anonymous multi-push multi-set for readability reasons.